keyboards/momokai/kei_6/info.json

```diff
@@ -0,0 +1,72 @@
+{
+    "manufacturer": "momokai",
+    "keyboard_name": "momokai/kei_6",
+    "maintainer": "peepeetee",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "rgblight": false,
+        "encoder_map": true
```

Comment:
This must be enabled at the keymap level.
```suggestion
```

Reply:
but why?

Reply:
done